.github/workflows/containers.yml

```diff
@@ -37,4 +40,4 @@ jobs:
         run: docker login -u $ACR_TOKEN_NAME -p ${{ secrets.ACR_APP_PUSH_TOKEN_PASSWORD }} $ACR_REGISTRY
 
       - name: Push App container
-        run: docker push $ACR_REGISTRY/public/ccf/app/${{ matrix.type }}:${{steps.tref.outputs.tag}}-${{ matrix.platform }}
+        run: docker push $ACR_REGISTRY/public/ccf/app/${{ matrix.type }}${{ matrix.run_js && '-js' || '' }}:${{ steps.tref.outputs.tag }}-${{ matrix.platform }}
```

Comment:
Argh, sorry, that's the part I couldn't test...

Reply:
No worries! I was going to say that we should test this workflow more often (#4591), but we wouldn't push to MCR then, so I guess that's inevitable.

Reply:
v_v